bigframes/core/compile/aggregate_compiler.py

```diff
@@ -416,7 +537,10 @@ def _(
 
 @compile_unary_agg.register
 def _(
-    op: agg_ops.AnyOp, column: ibis_types.Column, window=None
+    op: agg_ops.AnyOp,
+    column: ibis_types.Column,
+    window=None,
+    order_by: typing.Sequence[ibis_types.Value] = [],
```

Comment:
Hmm, this does clutter up the signature for just a single op. Not sure a better way though, maybe we will think of something later.

Reply:
Yeah, `order_by` is overwhelm to apply for all aggregate operations, while only `array_agg`, `string_agg` and `array_concat_agg` supports `order_by`.